sqldb/sqlutils.go

```diff
@@ -44,9 +44,9 @@ func sqlStr(s string) sql.NullString {
 	}
 }
 
-// sqlTime turns a time.Time into the NullTime that sql/sqlc uses when a time
+// SQLTime turns a time.Time into the NullTime that sql/sqlc uses when a time
 // can be permitted to be NULL.
-func sqlTime(t time.Time) sql.NullTime {
+func SQLTime(t time.Time) sql.NullTime {
```

Comment:
Looking forward to a future `sqlc` release once https://github.com/sqlc-dev/sqlc/pull/3279 is merged in.

Reply:
Oh nice!